SPEC.md

```diff
@@ -547,6 +566,55 @@ This facilitates "wildcard" matching and a variety of common usage patterns, lik
 For example, an AppImage containing a set of bash scripts might omit both "os" and "arch", and hence could be used as a dependency by a variety of different AppImages.
 Alternatively, an AppImage might specify a dependency with no image ID and no "version" label, and the image discovery mechanism could always retrieve the latest version of an AppImage
 
+### Pod Manifest Schema
+
+JSON Schema for the Pod Manifest
+
+```
+{
+
+    "acVersion": "0.2.0",
+    "acKind": "PodManifest",
+    "apps": [
+        {
+            "app": "example.com/reduce-worker",
+            "imageID": "sha512-..."
```

Comment:
This still seems like an oddity. It requires a priori knowledge of resolved images; there's no way to resolve an image based on its imageID...it implies images are already cached. So, the schema is bimodal here. That's _ok_ but food for thought. As long as Discovery and Dependency Matching are available to this manifest it would get my vote.